mypyc/test/test_exceptions.py

```diff
@@ -42,6 +43,7 @@ def run_case(self, testcase: DataDrivenTestCase) -> None:
                     if (fn.name == TOP_LEVEL_NAME
                             and not testcase.name.endswith('_toplevel')):
                         continue
+                    insert_uninit_checks(fn)
```

Comment:
What about cases when we have an uninitialized variable because of an exception? Example:

```py
def f() -> None:
    try:
        g()  # May raise
        x = 1
    except Exception:
        pass
    print(x)
```

Reply:
I'll add a test but I believe this should work, since we treat error handlers as successors when computing the CFG before exception-insertion now.

Reply:
Oh, darn it, no, it is trickier than that, since the failure can happen *first*.